src/Orleans.sln

```diff
@@ -271,4 +271,7 @@ Global
 	GlobalSection(NestedProjects) = preSolution
 		{0BB483EF-6401-424C-9C4F-5750B300C7FD} = {DD5D35C0-A5FE-41CC-A10D-CAAA37996479}
 	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		EnterpriseLibraryConfigurationToolBinariesPathV6 = packages\EnterpriseLibrary.TransientFaultHandling.6.0.1304.0\lib\portable-net45+win+wp8
```

Comment:
@jthelin I know you are passionate about manually removing version information from NuGet references. However, I haven't seen with my eyes yet where it would lead to a real issue. In the quite extensive 343 codebase nobody bothers to maintain such purity, and everything seems to work fine. Do you have a repro example of where having version info in references would lead to a problem?